doc/newfeat.texi

```diff
@@ -131,6 +131,16 @@ See the docstring for details. New user customization option
 @code{ess-assign-list} controls what assignment operators are cycled
 through.
 
+@item The ESS initialization process has been further streamlined.
+Users now should not need to @code{(require 'ess-site)} at all, as
```

Comment:
This is only valid for packaged ESS from MELPA, right?

Reply:
The news entry should be very clear about that. There are a range of other methods to install ESS and for those distributions people would still need to load ess-site.